kobo/apps/stripe/constants.py

```diff
@@ -22,14 +22,14 @@
 ORGANIZATION_USAGE_MAX_CACHE_AGE = timedelta(minutes=15)
 
 USAGE_LIMIT_MAP = {
-    'character': 'mt_characters',
+    'characters': 'mt_characters',
     'seconds': 'asr_seconds',
     'storage': 'storage_bytes',
     'submission': 'submission',
 }
 
 USAGE_LIMIT_MAP_STRIPE = {
-    'character': 'nlp_character',
+    'characters': 'nlp_character',
```

Comment:
You're going to find me annoying but, is it possible to also use `nlp_character`**s** (plural), to match other strings?

Reply:
This will be covered by https://www.notion.so/kobotoolbox/Switch-to-mt_characters-and-asr_minutes-in-code-stripe-2efeaf1173404bca8595e675440abb84